src/node.ls

```diff
@@ -29,7 +29,11 @@ module.exports = !(LiveScript) ->
     LiveScript <<<< events.EventEmitter.prototype
 
     require.extensions.'.ls' = (module, filename) ->
-        js = LiveScript.compile (fs.read-file-sync filename, 'utf8'), {filename, +bare, map: "embedded"} .code
+        file = (fs.read-file-sync filename, 'utf8')
+        if (filename.substr -7) is 'json.ls'
```

Comment:
@vendethiel can you give me some hints or instructions for the code style? I am a beginner when it comes to livescript and functional programming.